ledger/sandbox-classic/BUILD.bazel

```diff
@@ -366,6 +366,23 @@ server_conformance_test(
     ],
 )
 
+server_conformance_test(
+    name = "conformance-test-pruning-append-only",
+    lf_versions = [
+        "default",
+    ],
+    server_args = [
+        "--static-time",
```

Comment:
static-time and weak seeding are corner cases that we should try and avoid in a more general test.

Reply:
Done.